src/Raven.Server/ServerWide/Commands/GetRaftEtagCommand.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Raven.Server.ServerWide.Commands
+{
+    public class GetRaftEtagCommand : CommandBase
+    {
+        // This command is only to get a unique etag from the cluster (The RaftCommandIndex)
+        // We using it for recreating indexes on a recreated database
+        public GetRaftEtagCommand()
```

Comment:
I think in raft it is called index so maybe better name would be 'GetRaftIndexCommand' ?

Reply:
done